demo/src/layout/PropsTable.js

```diff
@@ -92,13 +92,8 @@ export default withStyles((theme) => ({
       paddingRight: "12px",
     },
   },
-  // description: {
-  //   fontSize: "16px",
-  //   color: theme.colors.black,
-  //   lineHeight: "1.5",
-  // },
   label: {
-    color: theme.colors.gray500,
+    color: theme.colors.gray600,
```

Comment:
if we are going to update this value, lets use the new values instead of legacy. So `theme.colors.gray['600']`